tests/main/views/test_g_cloud.py

```diff
@@ -29,7 +29,7 @@ def test_posting_lot_redirects_to_search_with_lot(self, lot):
         )
 
         assert res.status_code == 302
-        assert res.location.endswith(f"/g-cloud/search?lot={lot}")
+        assert res.location.endswith("/g-cloud/search" + f"?lot={lot}" if lot else "")
```

Comment:
I usually wrap ternary statements in brackets because it's hard to remember what the order of precedence is, but not important.